client/src/pages/DatabaseView.jsx

```diff
@@ -91,19 +107,25 @@ const DatabaseView = () => {
                     </select>
                 </div>
 
+                {feedbackMessage && (
+                    <div className={`mb-4 text-sm font-medium ${feedbackMessageType === 'success' ? 'text-green-600' : 'text-red-600'}`}>
+                        {feedbackMessage}
+                    </div>
+                )}
+
                 {/* Form for Add/Edit Operations */}
                 <div className="mb-4">
-                    <h2 className="text-lg font-semibold text-gray-800">{isEditing ? "Edit" : "Add New"} {dataType.slice(0, -1)}</h2>
+                <h2 className="text-lg font-semibold text-gray-800"> {isEditing ? `Editing user: "${formData.userName}" with ID "${editId}"` : `Add New ${dataType.slice(0, -1)}`}</h2>
+
                     <form onSubmit={handleSubmit} className="space-y-4">
                         {dataType === 'users' && (
                             <input
-                                name="username"
-                                value={formData.username || ''}
+                                name="userName"
+                                value={formData.userName || ''}
```

Comment:
its better to use the state here, since its nested, either have the key value pairs specified or split it out